src/main/java/org/terasology/gookeeper/system/VisitorSystem.java

```diff
@@ -28,6 +28,7 @@
 import org.terasology.entitySystem.systems.RegisterSystem;
 import org.terasology.entitySystem.systems.UpdateSubscriberSystem;
 import org.terasology.fences.ConnectsToFencesComponent;
+import org.terasology.gookeeper.Constants;
 import org.terasology.gookeeper.component.*;
```

Comment:
Should not have any star imports anywhere.
If intelliJ is automatically adding them then change the import threshold to 99 to stop it

Reply:
Yeah IntelliJ is doing that automatically.